### ISSUE TITLE
e-sparqlEndpoint cannot be compiled due to non-resolvable parent POM

### ISSUE THREAD

**Issue by jindrichmynarz** [UnifiedViews/Plugins#235]
e-sparqlEndpoint DPU (compiled from 1d62e50a02d304fe8e8ecdd405cab7c69c023cc4, `dpu/e-sparqlEndpoint` branch) cannot be compiled due to non-resolvable parent POM. Running `mvn install` returns the following error:

```
Non-resolvable parent POM: Failure to find eu.unifiedviews:uv-pom-dpu:pom:2.0.1 in http://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced and
'parent.relativePath' points at no local POM @ line 4, column 13 -> [Help 2]
```

It is probably the same issue as in #183 and #219.

**Comment by skrchnavy** [comment]
in commit 4856f202ad8ae3a92944040f89c1a23d1e414ea1 there is pom.xml without section for repositories (chicken/egg problem)

Need to add 
```
	<repositories>
		<repository>
			<id>maven.eea.sk</id>
			<name>maven.eea.sk-public</name>
			<url>http://maven.eea.sk/artifactory/public/</url>
		</repository>
	</repositories>
```
into the pom, please be inspired by for example https://github.com/UnifiedViews/Plugins/blob/master/e-filesDownload/pom.xml

**Comment by skrchnavy** [comment]
Branch with this code was deleted so this is invalid. Shall be fixed as a part of #178
